packages/xod-client-electron/src/view/containers/App.jsx

```diff
@@ -67,7 +67,7 @@ const defaultState = {
   workspace: '',
   downloadProgressPopup: false,
   downloadProgressPopupError: null,
-  sidebarPaneHeight: null,
+  sidebarPaneHeight: {},
```

Comment:
You use the term “panel” in most places. So, maybe to keep glossary shorter we’d better get rid of “pane”’s renaming it to panels?

Reply:
Oops, it's a trash code. Deleted it. Thank you :)